packages/msi/package.ps1

```diff
@@ -16,22 +19,24 @@ function info($msg) {  Write-Host $msg -f white }
 
 info "build msi package $version"
 # delete previous build
-Remove-Item .\mondoo.msi -ErrorAction Ignore
+Remove-Item ".\mondoo_${arch}.msi" -ErrorAction Ignore
```

Comment:
If the intend is to delete previous builds, for x64, should we try to remove the previous location too?

Reply:
kept both lines just in case, although in reality this runs in CI against an empty directory each time, so none of the files will ever be there